docs/examples/general/expressions/expr_type_promotions.ipynb

```diff
@@ -4,20 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p>Find it weird that 1, 2, 3 are scattered in different subsections</p><p> </p><p>some       images -&gt; too many spaces</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done</p>